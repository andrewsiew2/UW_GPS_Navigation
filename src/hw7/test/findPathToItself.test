# Find a path with Dijkstra's algorithm when the source and destination is itself

LoadGraph test testing.tsv

FindPath test NEVERMORE NEVERMORE
