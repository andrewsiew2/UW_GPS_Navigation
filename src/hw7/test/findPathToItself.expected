# Find a path with Dijkstra's algorithm when the source and destination is itself

loaded graph test

path from NEVERMORE to NEVERMORE:
total cost: 0.000